README.md

```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers of 2020
```

Comment:
Can we actually change this to Top 5 Albums? I think that would be a list worth seeing.

Reply:
Sure! I like that idea a lot.

Reply:
One last suggestion. Let's update the title to Top 5 Albums. That may be easier :)

Reply:
Top 5 albums of 2020? Or just in general?

Reply:
In general sounds good. Thanks!